### DISCUSSION TITLE
Disminuir el tamaño de los text presets de inputs en desktop

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1250] [Proposals]
### Goal

Mejorar la legibilidad de los textos 

### Context

En la creación del CMS estamos utilizando los text-presets de Mistica desktop, que tienen como text-preset más frecuente el equivalente a el text-preset-3 lo que consideramos demasiado grande (18px).

Nos gustaría poder bajar un preset (a text-preset-2) en todos los componentes de input para que podamos construir una UI más equilibrada.

### Impact

Creemos que este impacto no afectaría demasido porque no tocaríamos los presents de componentes como cards, si no solo aquellos usados para formularios. Además, consideramos que 16px ya se suficientemente grande para ser legible en desktop correctamente.

### Discussion Questions

-¿Es factible y tiene sentido el cambio?
-¿Sería mejor usar otra librería para hacer el CMS y dejar Mistica Desktop únicamente para hacer páginas de usuario?

### Do you have a visual proposal?

https://www.figma.com/file/7oGfCcnzS1okDr2xyRIE54/Entry-point?type=design&node-id=1716-38037&t=bv6nZWEVd1aEm6Lu-4
<img width="1313" alt="image" src="https://github.com/Telefonica/mistica-design/assets/95479574/666f46bf-65d8-41c0-8a8e-3666ceda14cb">

**Comment by yceballost** [comment]
Quizás debería existir ~ algo ~ para que Mística pueda adaptarse a interfaces más compactas. Algún tipo de config para que Mística cambie sus espacios y tamaños pensando en herramientas de backoffice.

No es la primera vez que lo pensamos, pero la tarea es grande.

**Reply by aweell** [reply]
Puede pasar que en el formulario no utilices items compactos y que aun así, un tamaño 18 para el value de un TextField sea excesivo no?

O crees que debería existir una correlación entre lo densa que sea la interfaz y el tamaño de values y labels de inputs?

Yo creo que aquí hay dos tracks diferentes: 

* Acomodar mejor los pesos de algunos componentes a un tamaño menor (16) para textFields y controles
* Proveer de versiones compactas para sobre todo items de formulario. Aquí incluso podría ajustarse el tamaño a 14 si fuera necesario.

**Comment by yceballost** [comment]
Con el ejercicio que hizo Héctor sobre el backoffice, consideramos que no es necesario el cambio de text-preset en desktop porque aparentemente parece que se puede resolver cambiando el layout de una pagina como el CMS
